src/module/endpoints/Endpoints.ts

```diff
@@ -0,0 +1,51 @@
+import {App} from "../../ManagementConsole";
+import "../../services/dmr/DmrService";
+import {EndpointConfigCtrl} from "./../endpoints/config/EndpointConfigCtrl";
+
+const module: ng.IModule = App.module("managementConsole.endpoints", []);
+
+
+module.config(($stateProvider: ng.ui.IStateProvider) => {
+
+  $stateProvider.state("edit-endpoint-config", {
+    parent: "root",
+    url: "endpoints/:profile/:endpointType/:endpointName/config",
```

Comment:
Ok, I get it.... but let's wait for this one as it is not essential now and we might still tune a bit this page. We can do the redirect at that time.